.drone.yml

```diff
@@ -2,6 +2,10 @@ workspace:
   base: /go
 
 pipeline:
+  clone:
+    image: plugins/git
+    tags: true
```

Comment:
I think there's some confusion somewhere. You are right, git clone all tags if set as true. However, I'm not using git tags. What I have is a `${DRONE_TAG}` envvar that contains my branch and my git sha.

Correct me if I'm wrong, but git clone doesn't create that envvar. [The link](https://github.com/appleboy/drone-scp/blob/master/Makefile#L22-L26) that you provided uses that envvar, but when [I tried to use it](https://github.com/ipedrazas/drone-helm/commit/5b91a1b48905b43d05311762635899d8a4b83043#diff-3216dfff0ed3e301453e6799e8c367e2R27), it was empty.

If you look at [the definition](https://github.com/drone/drone/blob/b7a0482c68fa5eac54e0e188bfed205efd685af8/agent/agent.go#L308) of the ${DRONE_TAG} envvar:

```
if w.Build.Event == model.EventTag {
	envs["DRONE_TAG"] = strings.TrimPrefix(w.Build.Ref, "refs/tags/")
}
```
`w.Build.Ref` is formed by `refs/heads/` and the branch name, like this: `refs/heads/mybranch`

So, I see where you're trying to go, but I do not use git tags, and the docker image tag that I want to create is formed by these two strings:

`TAG=${DRONE_BRANCH}-${DRONE_COMMIT_SHA:0:7}`